src/Store.js

```diff
@@ -0,0 +1,121 @@
+// import check from './check.js';
```

Comment:
### Optional
- [ ] Kindly delete the commented piece of code, it makes the codebase looks messy and difficult to understand